apps/examples/testcase/le_tc/kernel/tc_environ.c

```diff
@@ -87,6 +85,15 @@ static void tc_environ_setenv_getenv_unsetenv(void)
 	psz_getvalue = getenv("arv");
 	TC_ASSERT_EQ("getenv", psz_getvalue, NULL);
 
+	ret_chk = setenv(NULL, psz_value, 0);
+	TC_ASSERT_EQ("setenv", ret_chk, ERROR);
+
+	ret_chk = setenv(psz_name, NULL, 0);
```

Comment:
No, it was a mistake, Line 91, 92 need to be modified, it has to be different tc.